main.nf

```diff
@@ -544,9 +576,7 @@ if( params.aligner == 'bismark' ){
             """
         }
     }
-
-    ch_bismark_align_log_for_bismark_report
-     .join(ch_bismark_dedup_log_for_bismark_report)
+     ch_bismark_dedup_log_for_bismark_report
```

Comment:
This change will break functionality for regular users. This step is required to synchronise the logs for each sample from each step. Without it, random alignment logs from each sample will be mixed with random log sets from other samples.

Reply:
I think the only way to handle this is to have a second conditional join that adds on the alignment logs if we're not merging. This isn't very tidy, but I can't think of another way to handle both scenarios. Except just skipping the `bismark2report` step if we're merging.